ionic_user_interface/src/views/RequestGym.vue

```diff
@@ -5,6 +5,10 @@
         <strong>Route Maker</strong>
         <p>Climbing Gym Request Form</p>
         <br />
+        <p class="description">
```

Comment:
Font-color is black even in dark mode, cannot read at all

Reply:
good catch